### ISSUE TITLE
This Indexer master file indexer.oas2.json has an error (unspecified) when loading into postman. 

### ISSUE THREAD

**Issue by rfustino** [algorand/indexer#173]
(indexer.oas3.yml imports fine)

**Comment by winder** [comment]
@rfustino please provide more information about how to reproduce this problem. Thanks!

**Comment by ian-algorand** [comment]
@rfustino we're still waiting on replication steps please.

**Comment by rfustino** [comment]
Repro steps:

Clone repository
Start postman app. 
Hit the import button in the top left hand corner and upload indexer.oas2.json from your local clone.
Error message says Error while importing: format not recognized.

I noticed an update to this file on the develop branch 22 hours ago on 9/17 and the repo steps above apply to that branch as well, it still errors. indexer.oas.yml works fine. 

As for the second part of the issue https://github.com/algorand/go-algorand/issues/1496 ... 
regarding header key values...

...  Header key value names in the oas files for each end point. Right now the user needs to edit every end point at copy and paste the name of the name of the key or set up variables. Pasting in just the value for that key name would make it easier to use and defaulting the value to the sandbox would be a bonus." Example images attached for algod, but same can be done for indexer, just omitting a default token value for the key X-Indexer-API-Token

![Header-key-value-defaults](https://user-images.githubusercontent.com/4959925/93610992-b7602500-f99b-11ea-968d-f0c43f16e26c.png)
![Header-key-value-empty](https://user-images.githubusercontent.com/4959925/93610994-b7f8bb80-f99b-11ea-8a29-005545b9d66e.png)

See info on Parameters and Headers here:
https://learning.postman.com/docs/publishing-your-api/authoring-your-documentation/#parameters-and-headers

**Comment by winder** [comment]
Hi Russ, **indexer.oas3.yml** seems to work fine

![2020-09-18-114147_965x829_scrot](https://user-images.githubusercontent.com/125509/93617504-0f028e80-f9a4-11ea-827f-6bfe46e87a19.png)

**Comment by winder** [comment]
To clarify, this seems to be expected behavior. Postman does not support OpenAPI v2

https://learning.postman.com/docs/integrations/available-integrations/working-with-openAPI/

**Comment by rfustino** [comment]
Nice find! Hence why we provide v2 and v3!  Will create separate issue on user experience on import now.

**Comment by winder** [comment]
@rfustino great! Sounds like this is resolved now.
